test/performance/operf.go

```diff
@@ -0,0 +1,666 @@
+func (unit *LvmUnitMetrics) renderTemplate(metricsTemplate string) string {
+	tmpl, err := template.New("test").Parse(metricsTemplate)
+	if err != nil {
+		panic(err)
+	}
+	buf := new(bytes.Buffer)
+	err = tmpl.Execute(buf, unit)
+	if err != nil {
+		panic(err)
+	}
+	return buf.String()
+}
+
+// getPodMetrics gets the CPU and MEM metrics of pods
+func (unit *LvmUnitMetrics) getPodMetrics(start, end int64) {
+	query := unit.renderTemplate(CpuQueryTemplatePod)
+	metricsRes := unit.getMetrics(start, end, query)
+	if metricsRes == "" {
+		panic("CPU Metrics not found")
+	}
+	podCpu := unit.parseMetrics(metricsRes)
+
+	query = unit.renderTemplate(MemQueryTemplatePod)
+	metricsRes = unit.getMetrics(start, end, query)
+	if metricsRes == "" {
+		panic("MEM Metrics not found")
+	}
+	podMem := unit.parseMetrics(metricsRes)
+
+	for key := range podCpu {
+		unit.ObjectMetrics[key] = ObjectMetric{
+			Name: key,
+			Type: "Pod",
+			Cpu: Metric{
+				Min: podCpu[key].Min,
+				Max: podCpu[key].Max,
+				Avg: podCpu[key].Avg,
+			},
+			Mem: Metric{
+				Min: podMem[key].Min / Mebibyte,
+				Max: podMem[key].Max / Mebibyte,
+				Avg: podMem[key].Avg / Mebibyte,
+			},
+		}
+	}
+}
+
+// getContainerMetrics gets the CPU and MEM metrics of containers
+func (unit *LvmUnitMetrics) getContainerMetrics(start, end int64) {
+	query := unit.renderTemplate(CpuQueryTemplateContainer)
+	metricsRes := unit.getMetrics(start, end, query)
+	if metricsRes == "" {
+		panic("CPU Metrics not found")
+	}
+	containerCpu := unit.parseMetrics(metricsRes)
+
+	query = unit.renderTemplate(MemQueryTemplateContainer)
+	metricsRes = unit.getMetrics(start, end, query)
+	if metricsRes == "" {
+		panic("MEM Metrics not found")
+	}
+	containerMem := unit.parseMetrics(metricsRes)
+
+	for key := range containerCpu {
+		unit.ObjectMetrics[key] = ObjectMetric{
+			Name: key,
+			Type: "Container",
+			Cpu: Metric{
+				Min: containerCpu[key].Min,
+				Max: containerCpu[key].Max,
+				Avg: containerCpu[key].Avg,
+			},
+			Mem: Metric{
+				Min: containerMem[key].Min / Mebibyte,
+				Max: containerMem[key].Max / Mebibyte,
+				Avg: containerMem[key].Avg / Mebibyte,
+			},
+		}
+	}
+}
+
+// getAllMetrics gets the metrics of pods and containers
+func (unit *LvmUnitMetrics) getAllMetrics(start, end int64) {
+	unit.Start = time.Unix(start, 0)
+	unit.End = time.Unix(end, 0)
+
+	unit.ObjectMetrics = map[string]ObjectMetric{}
+	unit.getPodMetrics(start, end)
+	unit.printMetrics()
+
+	unit.ObjectMetrics = map[string]ObjectMetric{}
+	unit.getContainerMetrics(start, end)
+	unit.printMetrics()
+}
+
+// printMetrics prints the CPU and MEM metrics
+func (unit *LvmUnitMetrics) printMetrics() {
+	for _, u := range unit.ObjectMetrics {
+		fmt.Println(strings.Repeat("-", 80))
+		fmt.Printf("Report for %s %s's %s %s between %s and %s\n", unit.Name, unit.WorkloadType, u.Type, u.Name, unit.Start, unit.End)
+		fmt.Printf("\t CPU (min|max|avg) seconds: % 10.4f | % 10.4f | % 10.4f\n", u.Cpu.Min, u.Cpu.Max, u.Cpu.Avg)
```

Comment:
Not sure if the unit for `container_cpu_usage_seconds_total` is in seconds. The prometheus metrics shows different unit in the openshift. 
![Screenshot from 2022-06-13 15-00-25](https://user-images.githubusercontent.com/9363998/173323985-6b622e13-af9c-4dc5-beed-946b0772cdb1.png)

Reply:
This is what i get after running the exact query

![Screenshot from 2022-06-13 15-21-10](https://user-images.githubusercontent.com/22993933/173328143-460a0790-4fd0-4326-9922-fa7470d5fb2b.png)